docs/qcfractal/source/changelog.rst

```diff
@@ -15,6 +15,37 @@ Changelog
 .. Bug Fixes
 .. +++++++++
 
+
+0.13.0 / 2020-01-15
+-------------------
+
+New Features
+++++++++++++
+- (:pr:`541`) Managers can now accept multiple tags. Tasks are pulled first in order of tag, then priority, then creation time.
+- (:pr:`544`) Adds backup/restore commands to the QCFractal CLI to allow for easier backup and restore options.
+
+Enhancements
+++++++++++++
+- (:pr:`507`) Automatically adds collection molecules in chunks if more than the current limit needs to be submitted.
+- (:pr:`515`) Conda environments now correspond to docker images in all deployed cases.
+- (:pr:`524`) The ``delete_collection`` function was added to ``qcportal.FractalClient``.
+- (:pr:`530`) Adds the ability to specify cores per rank for node-parallel tasks in QCEngine.
+- (:pr:`507`) Adds a formatting and lint check to CI during pull requests.
+- (:pr:`535`) Allows dftd3 to be computed for all stoichiometries rather than just defaults.
+- (:pr:`542`) Important: TaskRecord.base_result is now an ObjectId (int or str), and no more a ``DBRef``. So, code that uses ``my_task.base_result.id`` should change to simply use ``my_task.base_result``.
```

Comment:
Can we mark this as a Breaking Change?

Reply:
This seemed minor and internal enough that I was going to gloss over it as this should not effect user space. I don't think we have marked anything as so yet. Do you have a recommended way of doing this?

Reply:
Does it break our fractal examples for running and checking on calculations?

Reply:
It does not.

Reply:
Then I'm happy with the changelog as is.